src/app/components/ATIAnalytics/canonical/index.tsx

```diff
@@ -55,13 +56,16 @@ const CanonicalATIAnalytics = ({
     if (!isOperaProxy()) sendBeacon(atiPageViewUrl, reverbBeaconConfig);
   }, [atiPageViewUrl, reverbBeaconConfig]);
 
+  const liteSiteReverbParams =
```

Comment:
```suggestion
  const liteSiteReverbURL =
```

Reply:
Or perhaps we should be really explicit

```suggestion
  const liteSiteReverbPaegViewURL =
```